src/theme/foundations/typography.ts

```diff
@@ -238,15 +243,15 @@ const typography: Typography = {
     caption: fontWeightValues["regular"],
     default: fontWeightValues["regularImportant"],
     heading: {
-      display1: fontWeightValues["thin"],
-      heading1: fontWeightValues["thin"],
-      heading2: fontWeightValues["thin"],
-      heading3: fontWeightValues["light"],
-      heading4: fontWeightValues["regular"],
-      heading5: fontWeightValues["medium"],
-      heading6: fontWeightValues["medium"],
-      heading7: fontWeightValues["medium"],
-      heading8: fontWeightValues["medium"],
+      display1: fontWeightValues["lightImportant"],
```

Comment:
Do you _have_ to create these new `important` variables? If you set it as `light: "300 !important"`, will it override other styles where `light` is used? If not, I think it's fine to use `!important` in the existing variables.

Reply:
The only reason we are doing this is because of the annoying overrides coming from the Header app using DS v2. And logically, we do want the styles from v3 to be _THE_ styles and not be overridden by what is coming out of the Header and Footer. Having said that, yes, I also now agree that we could put the `!important` right in the original variables. I will do that an push the updates for another review.